pkgconfig/CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ set(ENCLAVE_CLIBS_1
   -Wl,-z,now
   -L\${libdir}/openenclave/enclave
   -loeenclave
+  -loecryptombed
```

Comment:
Would the sample cmake files need to be updated to link with oecryptombed?

Reply:
The samples use pkg-config, so updating this file automatically updates them.